docs/tutorial.md

```diff
@@ -0,0 +1,351 @@
+
+## Shop like a hipster!
+
+In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase" everything you need for your hipster lifestyle using a mock credit card number.
+
+## Run the load generator
+
+In another browser tab, navigate to the load-generator URL, from which you can simulate users interacting with the application to generate traffic. For this application, values like 100 users with a "hatch rate" of 2 (spawn 2 users per second) are reasonable.
+
+From here, you can explore how the application was deployed, and you can use the  
+navigation menu to bring up other GCP tools.
+
+## Learn Ops Management
+
+### Ops Management Overview
+
+As the cloud-native microservice architecture, which promises scalability and flexibility benefits, gets more popular, developers and administrators need tools that can work across cloud-based distributed systems.
+
+Ops Management provides products for both developers and administrators; this section introduces the products and their general audiences.  The tools are covered in more detail later.
+
+Application developers need to be able to investigate the cause of problems in applications running in distributed environments, and in this context, the importance of** Application Performance Management (APM)** has increased. Ops Management provides 3 products for APM:
+
+-  Cloud Trace
+-  Cloud Profiler
+-  Cloud Debugger
+
+Similarly, cloud-native, microservice-based applications complicate traditional approaches used by administrators for monitoring system health: it's harder to observe your system health when the number of instances is flexible and the inter-dependencies among the many components are complicated. In the last few years, **Site Reliability Engineering (SRE)** has become recognized as a practical approach to managing large-scale, highly complex, distributed systems. Ops Management provides the following tools that are useful for SRE:
+
+-  Cloud Monitoring
+-  Cloud Logging
+-  Cloud Error Reporting
+
+You can find the Ops Management products in the navigation panel on the GCP Console:
+
+## The Stackdriver APM products: Trace, Profiler, and Debugger
+
+Next, learn about the APM products!
+
+## Cloud Trace Overview
+
+Stackdriver Trace ([documentation](https://cloud.google.com/trace/docs/)) enables developers to see distributed traces that visually expose latency bottleneck in requests. Developers instrument application code to collect trace information. You can also include environmental information in traces and trace information can be included in Stackdriver Logging logs. The Trace UI can then pull relevant log events into the trace timelines. 
```

Comment:
Stackdriver Trace is now Cloud Trace, similarly Cloud Monitoring and Cloud Logging. Please fix the branding names accordingly throughout.

Reply:
Okay, changed the branding names!